src/pages/SubmissionDetails.js

```diff
@@ -39,26 +40,32 @@ export const SubmissionDetails = ({ submissionId }) => {
     loadRunsData()
   })
 
-  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, runsData)
+  const sortedPreviousRuns = orderBy(sort.field, sort.direction, runsData)
 
   const firstPageIndex = (pageNumber - 1) * itemsPerPage
   const lastPageIndex = firstPageIndex + itemsPerPage
   const paginatedPreviousRuns = sortedPreviousRuns.slice(firstPageIndex, lastPageIndex)
 
+  const header = useMemo(() => {
+    const breadcrumbPathObjects = [
+      {
+        label: 'Submission History',
+        path: 'submission-history'
+      },
+      {
+        label: `Submission ${submissionId}`
+      }
+    ]
+
+    return h(HeaderSection, { breadcrumbPathObjects, title: 'Submission Details' })
+  }, [submissionId])
+
   const rowWidth = 100
   const rowHeight = 200
-  return h(Fragment, [
-    headerBar(),
-    div({ style: { margin: '4em' } }, [
-      div({ style: { display: 'flex', marginTop: '0.5rem', justifyContent: 'space-between' } }, [
-        h1(['Submission Details']),
-        h(ButtonOutline, {
-          onClick: () => Nav.goToPath('root')
-        }, ['Submit a new workflow'])
-      ]),
-      div([
-        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`])
-      ]),
+  return div({ id: 'submission-details-page' }, [
+    Navbar('SUBMIT WORKFLOWS WITH CROMWELL'),
```

Comment:
I'd give it even odds that the inconsistent headers in the mocks are an error, or that we'll decide to make it consistent later. For now, though, flexibility FTW.